man/coreChecklist.Rd

```diff
@@ -0,0 +1,32 @@
+% Auto-generated: do not edit by hand
+\name{coreChecklist}
+\alias{coreChecklist}
+\title{Checklist component}
+\description{
+Checklist is a component that encapsulates several checkboxes. The values and labels of the checklist is specified in the `options` property and the checked items are specified with the `values` property. Each checkbox is rendered as an input with a surrounding label.
+}
+\usage{
+coreChecklist(..., labelClassName='', style=NULL, inputClassName='', inputStyle=list(), labelStyle=list(), options=list(), className=NULL, values=NULL, id=NULL)
+}
+\arguments{
+\item{labelClassName}{The class of the <label> that wraps the checkbox input
+ and the option's label}
+
+\item{style}{The style of the container (div)}
+
+\item{inputClassName}{The class of the <input> checkbox element}
+
+\item{inputStyle}{The style of the <input> checkbox element}
+
+\item{labelStyle}{The style of the <label> that wraps the checkbox input
+ and the option's label}
+
+\item{options}{An array of options}
+
+\item{className}{The class of the container (div)}
+
+\item{values}{The currently selected value}
+
+\item{id}{}
+}
+    
```

Comment:
There is inconsistent whitespace at the end of these `.Rd` files, as well as trailing whitespace on some of the blank lines above. Can we get rid of these?